### PR TITLE
chore: Build Script Cleanup + Windows Support

### PR DESCRIPTION
**Pull request by ArcaneDisgea**
Using cross-env to address #13 and add windows build support as well as adjusting gitignore to allow us to include the ``demo.html``.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -81,7 +81,7 @@ typings/
 .nuxt
 
 # rollup.js default build output
-dist/
```
Comment (ayyaruq) [inline .gitignore]: Don't ever add dist to repo. If a file in here is really needed, it should be located elsewhere.

**Review by ayyaruq — CHANGES_REQUESTED** [review]
Revert the gitignore change and move the demo.html elsewhere imo. Dist should never be committed. This'll need to be squash merged or rebased to avoid polluting history now too.

File: .gitignore (hunk 1 above)
Comment (ghost) [reply .gitignore]: maybe `examples`? That way they can `yarn build:dev`, and then go to `examples/demo.html` and have it refer to `../dist/...` in the path?

Comment (ArcaneDisgea) [reply .gitignore]: Should be good to go now.

File: example/demo.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8"/>
+        <title>Inline template test</title>
+        <link href="https://fonts.googleapis.com/css?family=Mada:400,500&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Pathway+Gothic+One&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="https://use.typekit.net/bbh8fwb.css">
+        <link rel="stylesheet" href="https://use.typekit.net/xwu6uop.css">
+        <link rel="stylesheet" type="text/css" href="../dist/vue-xivtooltips.css" />
```
Comment (ghost) [inline example/demo.html:10]: I would have missed this. Good catch.

Comment (ayyaruq) [reply example/demo.html:10]: 👀

**Review by ayyaruq — APPROVED** [review]
Noice, toight

**Pull request merged**